### PR TITLE
Feature/milestone 2

### PR DESCRIPTION
**Pull request by AshakaE**
### In this milestone, I did,
- Create the components directory
- Create a Display component with proptype validation
- Create a Button component
- Create a ButtonsPanel component 
- Display the raw component to the webpage

### CODE REVIEW

**Comment by addod19** [comment]
## Status: Invalid!!! :red_circle: 

Hi @AshakaE, I will have to mark this project as invalid because the linter is not set up to check for errors. Kindly fix that and submit another code review

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => {
+  const { name } = props;
```
Comment (Forison) [inline src/components/Button.js]: **[Optional]**
We can destructure `name` at the argument level, that way we avoid line 5.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes required❌.

# 👋  @AshakaE 

#### Good job on this one so far, the pull request is very professional, Good job on the use of React fragment and validating all your props💪 .

* [x] App.js renders the Display and ButtonPanel components ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png)). Currently you are only rendering Display.jsx.

#### That is it kindly make the change and submit a re-review🤗.

#### Happy coding😎 

Addo Forison

File: src/components/App.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.App {
```
Comment (Forison) [inline src/components/App.css]: * [x] Kindly remove this code, there are no requirement for styling, at least not for now. unrelated code must be removed.

**Review by Forison — CHANGES_REQUESTED** [review]

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @AshakaE, the comment from the previous review was not addressed. I will point it out here as well so we can address it :slightly_smiling_face: Kindly check the comments below:
- The App.js file should render 2 components, ie. the Display component which we have currently, and the ButtonPanel component which needs to be rendered in the App component

Once this is fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @AshakaE, congratulations on this milestone :fireworks: Now the ButtonPanel is rendered in the App component :1st_place_medal: 

I hereby approve the project :smile: 
Kindly go ahead and merge into the development branch, do not forget to submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**